hackclub.com.yaml

```
@@ -415,6 +415,10 @@ assets:
   ttl: 1
   type: CNAME
   value: cname.vercel-dns.com.
+astrohacks:
+  - ttl: 1
+    type: CNAME
+    value: www.mmgraphix.net
```

```suggestion
    value: www.mmgraphix.net
```

This will need to end in a `.`